oaknational/Oak-Web-Application
# feat: release new plan a lesson page [LESQ-831]

## Description

Music year: 1980

- remove old plan a lesson page and tests
- move new plan a lesson page to correct url
- remove noIndex and noFollow on new plan a lesson page
- the postForm used in cms components wasn't complying with a11y requirements due to missing `aria-describedBy` field, so I had to make this optional on the newsletter form

## How to test

1. Go to https://deploy-preview-2603--oak-web-application.netlify.thenational.academy
2. Click on 'Plan a lesson' in the hamburger menu
3. You should see the new plan a lesson page

## Screenshots

How it used to look (delete if n/a):

<img width="700" alt="Screenshot 2024-07-09 at 08 57 57" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/d166af13-c919-4700-8ff2-944779f753d3">

How it should now look:

<img width="700" alt="Screenshot 2024-07-09 at 08 57 45" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/27bbb507-7b46-4015-b7f2-d75c6ea69eee">

## Checklist
- [ ]  The new lesson planning page appears at /lesson-planning
- [ ]  The new lesson planning page is set to “index, follow”
